services/logs2notifications/internal/handler/email_events.go

```diff
@@ -115,11 +122,11 @@ func (h *Messaging) processEmailTemplates(notification *Notification) (string, s
 <li><a href="{{.Route}}">{{.Route}}</a></li>
 {{range .Routes}}{{if ne . $.Route}}<li><a href="{{.}}">{{.}}</a></li>
 {{end}}{{end}}</ul>`
-		plainTextTpl = `[{{.ProjectName}}] {{.BranchName}}{{ if ne .ShortSha "" }} ({{.ShortSha}}){{end}} Build {{.BuildName}} complete. {{if ne .LogLink ""}}[Logs]({{.LogLink}}){{end}}
+		plainTextTpl = `[{{.ProjectName}}] {{.BranchName}}{{ if ne .ShortSha "" }} ({{.ShortSha}}){{end}} Build {{.BuildName}} ` + msg + `. {{if ne .LogLink ""}}[Logs]({{.LogLink}}){{end}}
 {{.Route}}
 {{range .Routes}}{{if ne . $.Route}}{{.}}
 {{end}}{{end}}`
-		subject += fmt.Sprintf("[%s] %s Build %s complete.",
+		subject += fmt.Sprintf("[%s] %s Build %s completed.",
```

Comment:
The subject line here should probably also match the msg ie completed/completed with warnings?

Reply:
ah yep nice